### ISSUE TITLE
Add a check sign (✔️) once the settings finish saving

### ISSUE THREAD

**Issue by radu-themeisle** [Codeinwp/wp-product-review#398]
When the options are saved, the loading indicator next to the button vanishes.
There's no feedback to show the process has finished.

I was thinking we could just add a simple "✔️" once the process is done.

**Comment by contactashish13** [comment]
@radu-themeisle I'll keep this on hold till we get more requests for this. For the time being, it doesn't seem to be a problem for users.

**Comment by radu-themeisle** [comment]
I don't agree to wait for further request as this is rather something related to product design improvement, but I agree we can focus for the moment on more important issues and keep this in the backlog.

**Comment by contactashish13** [comment]
Closing this as no one seems to have asked for this
